File: Leetcode/javascript/sol2089.js

// https://leetcode.com/problems/find-target-indices-after-sorting-array/description/
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var targetIndices = function(nums, target) {
        nums.sort((a, b) => a - b);
        var size = nums.length;
        var rslt = [];
        console.log(nums);
        for(let i = 0; i < size; i++){
            if (nums[i] == target){
                rslt.push(i);
            }
        }
        return rslt;

};

File: Leetcode/cpp/sol2089.cpp

//https://leetcode.com/problems/find-target-indices-after-sorting-array/

class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {

        sort(nums.begin(), nums.end());
        int size = nums.size();
        vector<int> rslt;
        for(auto i = 0; i < nums.size(); i++){
            if (nums[i] == target){
                rslt.push_back(i);
            }
        }
        return rslt;

    }
};